public/forms/builder/meta.ttl

```diff
@@ -71,6 +71,12 @@ display:textArea
   core:uuid "2e114946-3f3e-43ee-af16-d7c7255977e3" ;
   skos:inScheme <http://lblod.data.gift/concept-schemes/c5a91bd7-3eb5-4d69-a51b-9bac6bf345f6> ;
   skos:prefLabel  "Text area"@en .
+display:richTextEditor
+  rdf:type  rdfs:Class , skos:Concept ;
+  ext:displayType "richTextEditor" ;
+  ext:usesConceptScheme "false"^^xsd:boolean ;
+  # skos:inScheme <http://lblod.data.gift/concept-schemes/c5a91bd7-3eb5-4d69-a51b-9bac6bf345f6> ; This is commented as it is experimental at this moment of time
```

Comment:
As comment says. Experimental. Can be used directly in the ttl but the builder will not show the option